allennlp/commands/train.py

```diff
@@ -136,6 +136,8 @@ def train_model_from_file(parameter_filename: str,
         If ``True``, we will try to recover a training run from an existing serialization
         directory.  This is only intended for use when something actually crashed during the middle
         of a run.  For continuing training a model on new data, see the ``fine-tune`` command.
+    force : ``bool``, optional (default=False)
+        If ``True`` and a serialization directory exists, we will overwrite the output directory.
```

Comment:
Minor nit: maybe `If ``True``, we will overwrite the serialization directory if it already exists.`

Reply:
Got it. Happy holiday.